tools/bin/standard_test_pr.sh

```diff
@@ -7,4 +7,5 @@ set -e
 assert_root
 
 ./gradlew --no-daemon --scan -x generateProtocolClassFiles \
-  :airbyte-integrations:connectors:source-github-singer:standardSourceTestPython
+  :airbyte-integrations:connectors:source-github-singer:standardSourceTestPython \
+  :airbyte-integrations:connectors:source-google-sheets:standardSourceTestPython
```

Comment:
any ratelimiting on the google sheets API that we should be worried about?

Reply:
nothing to be worried about, it's 500 requests per 100 seconds. If we have problems with this, it'll mean our build is blazing fast ;)